### PR TITLE
[semver-major] Bugsnag filter, remove wrap-async-context, heap snapshots

### PR DESCRIPTION
**Pull request by SomeHats**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -58,11 +58,8 @@ export default (options) => {
     releaseStage: process.env.AWS_ENV || 'local',
     notifyReleaseStages: ['development', 'production', 'staging'],
     projectRoot: '/app',
-    metaData: {
-      get requestId() {
-        return (context() || {}).id;
-      },
-    },
+    filters: config.bugsnagFilters,
```
Comment (jgwmaxwell) [inline src/index.js]: Can we merge these? I think we should always insist on `password` and `card`?

**Comment by jgwmaxwell** [comment]
Only one comment on this one...

**Comment by dotbear** [comment]
I approve of this because it has `winston` as the only word I know.

Approved

**Comment by leemhenson** [comment]
Approved.

[Approved with PullApprove](https://pullapprove.com/musicglue/mg-express/pull-request/16/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=musicglue/mg-express)

**Comment by jgwmaxwell** [comment]
Approved

[Approved with PullApprove](https://pullapprove.com/musicglue/mg-express/pull-request/16/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=musicglue/mg-express)

**Pull request merged**